### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xIonRod**
- Added info about LunCo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 #### Godot 3.2+
 
 - [Broken Seals](https://github.com/Relintai/broken_seals) - A work-in-progress third person action RPG with multiplayer support (needs custom engine modules).
+- [LunCo](https://github.com/lun-co/lunco-game) – A work-in-progress Lunar Colony digital twin simulation game inspired by Kerbal Space Program, SimCity, Factorio.
```
Comment (Calinou) [inline README.md]:
```suggestion
- [LunCo](https://github.com/lun-co/lunco-game) - A work-in-progress Lunar Colony digital twin simulation game inspired by Kerbal Space Program, SimCity, Factorio.
```

**Comment by Calinou** [comment]
I'd wait until the project is further developed to consider merging this. I'd prefer keeping a quality standard for projects listed on this repository, especially as time goes on – we can't have the list grow forever without bound.

**Comment by 0xIonRod** [comment]
Ok!

Could you share any KPI that show that the game is mature enough to be added?

**Comment by Calinou** [comment]
> Could you share any KPI that show that the game is mature enough to be added?

I don't think there's an objective metric threshold for this. That said, projects with several contributors and an active player community are more likely to be added.

**Comment by 0xIonRod** [comment]
Thank you for response!

We already have some traction.

I hope in a week or so to have strong enough community.

I'll let you know, thank you one more time

**Pull request closed without merging**